test/test_integration_variable.py

```diff
@@ -0,0 +1,225 @@
+                       ['policy', 'replace', '-b', 'root', '-f', self.environment.path_provider.get_policy_path("variable")])
+        Utils.set_variable(self, 'variablespecialchars', '"[]{}#@^&<>~\/''\/?\;\';\'"')
+        output = Utils.get_variable(self, 'variablespecialchars')
+        self.assertEquals(output.strip(), '"[]{}#@^&<>~\/''\/?\;\';\'"')
+
+    @integration_test
+    def test_variable_get_variable_has_spaces_returns_variable_value(self):
+        self.invoke_cli(self.cli_auth_params,
+                       ['policy', 'replace', '-b', 'root', '-f', self.environment.path_provider.get_policy_path("variable")])
+        Utils.assert_set_and_get(self, "some variable with spaces")
+
+    @integration_test
+    def test_unknown_variable_raises_not_found_error(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'get', '-i', 'unknown'], exit_code=1)
+        self.assertRegex(output, "404 Client Error: Not Found for url:")
+
+    @integration_test
+    def test_cli_can_batch_get_multiple_variables(self):
+        policy, variables = Utils.generate_policy_string(self)
+        file_name=os.path.join(tempfile.gettempdir(), os.urandom(24).hex())
+        with open(file_name, 'w+b') as temp_policy_file:
+            temp_policy_file.write(policy.encode('utf-8'))
+            temp_policy_file.flush()
+
+            Utils.load_policy(self, temp_policy_file.name)
+        value_map = {}
+        for variable in variables:
+            value = uuid.uuid4().hex
+            Utils.set_variable(self, variable, value)
+            value_map[variable] = value
+
+        batch_result_string = Utils.get_variable(self, *variables)
+        batch_result = json.loads(batch_result_string)
+
+        for variable_name, variable_value in value_map.items():
+            self.assertEquals(variable_value, batch_result[variable_name])
+
+        os.remove(file_name)
+
+    @integration_test
+    def test_batch_existing_and_nonexistent_variable_raises_error(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'get', '-i', 'one/password,unknown'], exit_code=1)
+        self.assertIn("404 Client Error", output)
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_get_short_help_returns_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'get', '-h'], exit_code=0)
+        self.assertIn("usage: variable", output)
+
+    # TODO This will need to be changed when UX is finalized
+    @integration_test
+    def test_subcommand_get_long_help_returns_help(self):
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['variable', 'get', '--help'], exit_code=0)
+        self.assertIn("usage: variable", output)
+
+    @integration_test
+    def test_subcommand_get_help_returns_help(self):
```

Comment:
The name should be different 
test_variable_get_without_subcommand or something like this. (need to update test plan as well, if we change this)

Reply:
updating to `test_variable_get_without_id_returns_help`